TPGSup/TPG300_switching_functions.db

```diff
@@ -0,0 +1,92 @@
+record(bo, "$(P)FUNCTION:$(Q):SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)FUNCTION:$(Q):DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+record(fanout, "$(P)FUNCTION:$(Q):RB:FAN")
```

Comment:
If `RB` stands for readback, I think it would be worth following the [ISIS PV naming scheme](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#signal-qualifiers). If we must use a "readback" qualifier then `RBV` is preferred over `RB`. But most devices omit the `:RBV` pv and just follow the `VALUE`, `VALUE:SP`, `VALUE:SP:RBV` naming scheme.